packages/ua-devtools/src/oapp/config.ts

```diff
@@ -240,84 +240,126 @@ export const configureReceiveLibraryTimeouts: OAppConfigurator = withOAppLogger(
+        const setConfigsByEndpointAndLibrary: OmniPointMap<Map<OmniAddress, SetConfigParam[]>> = new OmniPointMap()
+
+        for (const {
+            vector: { from, to },
+            config,
+        } of graph.connections) {
+            if (config?.sendConfig?.executorConfig == null && config?.sendConfig?.ulnConfig == null) {
+                logger.verbose(`executorConfig and ulnConfig not set for ${formatOmniVector({ from, to })}, skipping`)
+                continue
+            }
 
-        if (config.sendConfig.executorConfig != null) {
-            // We ask the endpoint SDK whether this config has already been applied
-            //
-            // We need to ask not for the final config formed of the default config and the app config,
-            // we only need to check the app config
-            const hasExecutorConfig = await endpointSdk.hasAppExecutorConfig(
-                from.address,
-                currentSendLibrary,
-                to.eid,
-                config.sendConfig.executorConfig
+            const oappSdk = await createSdk(from)
+            const endpointSdk = await oappSdk.getEndpointSDK()
+            const currentSendLibrary = config.sendLibrary ?? (await endpointSdk.getSendLibrary(from.address, to.eid))
+            assert(
+                currentSendLibrary !== undefined,
+                'sendLibrary has not been set in your config and no default value exists'
             )
 
-            if (!hasExecutorConfig) {
-                const newSetConfigs: SetConfigParam[] = await endpointSdk.getExecutorConfigParams(currentSendLibrary, [
-                    { eid: to.eid, executorConfig: config.sendConfig.executorConfig },
-                ])
-
-                // Updates map with new configs for that OApp and Send Library
-                const setConfigsByLibrary = setConfigsByEndpointAndLibrary.getOrElse(from, () => new Map())
-                const existingSetConfigs = setConfigsByLibrary.get(currentSendLibrary) ?? []
-                setConfigsByEndpointAndLibrary.set(
-                    from,
-                    setConfigsByLibrary.set(currentSendLibrary, [...existingSetConfigs, ...newSetConfigs])
+            if (config.sendConfig.executorConfig != null) {
+                // We ask the endpoint SDK whether this config has already been applied
+                //
+                // We need to ask not for the final config formed of the default config and the app config,
+                // we only need to check the app config
+                const hasExecutorConfig = await endpointSdk.hasAppExecutorConfig(
+                    from.address,
+                    currentSendLibrary,
+                    to.eid,
+                    config.sendConfig.executorConfig
                 )
+
+                logger.verbose(
+                    `Checked executor configuration for ${formatOmniVector({ from, to })}: ${printBoolean(hasExecutorConfig)}`
+                )
```

Comment:
change 1

Reply:
Despite[ this test ](https://github.com/LayerZero-Labs/devtools/blob/aae5fa2cd7a206968e46857a4e592000d5377db3/tests/ua-devtools-evm-hardhat-test/test/oapp/config.test.ts#L757-L767) having a `config.sendConfig.executorConfig`, this log doesn't appear in the test build run. Perhaps I am checking the wrong test?

I would like to confirm this log and the `updatedConfigListCsv` log in the test build run before merging this PR

Reply:
The above is resolved -- once I expose the test networks on localhost, deploy and wire them with `--log-level verbose`, I am able to see these logs